includes/pages/user_myshifts.php

```diff
@@ -85,7 +91,9 @@ function user_myshifts()
                         . ' from ' . $shift->start->format('Y-m-d H:i')
                         . ' to ' . $shift->end->format('Y-m-d H:i')
                         . ' with comment ' . $comment
-                        . '. Freeloaded: ' . ($freeloaded ? 'YES Comment: ' . $freeloaded_comment : 'NO')
+                        . '. Freeloaded' . ($freeloaded_by
+                            ? ' by ' . User_Nick_render($freeloaded_by, true) . ' with Comment: ' . $freeloaded_comment
```

Comment:
```
Exception: Attempt to read property "displayName" on int
```


`$freeloaded_by` is just the user ID so the corresponding user must be found here

Reply:
Technically its `findOrFail` but whatever